eleventy.config.js

```diff
@@ -1,24 +1,39 @@
 const path = require('path');
+
 const fs = require('fs-extra');
 const { DateTime } = require('luxon');
-
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
 const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
 const xmlFiltersPlugin = require('eleventy-xml-plugin');
+const Nunjucks = require('nunjucks');
 
 const slugify = require('./libs/slugify');
-const nunjucksEnvironment = require('./libs/templates');
```

Comment:
Not sure what that was but with or without produces the same output 🤷

Reply:
On Ew the markdown instance was re-used so it was externalized and I did the same thing with the liquidJS template instance to cut down on the code in this file. 

You could keep all the Nunjucks stuff in that file (including the DOMpurify addition) to keep this more "just config" as much as possible. But up to you.